react-responsive-modal/src/lib/focusTrapJs.ts

```diff
@@ -12,16 +12,17 @@ export const candidateSelectors = [
   '[contenteditable]:not([contenteditable="false"])',
 ];
 
-function isHidden(node: any) {
+function isVisible(node: any) {
   // offsetParent being null will allow detecting cases where an element is invisible or inside an invisible element,
   // as long as the element does not use position: fixed. For them, their visibility has to be checked directly as well.
   return (
-    node.offsetParent === null || getComputedStyle(node).visibility === 'hidden'
+    node.offsetParent !== null ||
+    getComputedStyle(node).visibility === 'visible'
```

Comment:
I need to think deeper on this change - I'm pretty sure I misunderstood original implementation...

Reply:
That was a mistake I reverted.